gaussian_renderer/__init__.py

```diff
@@ -14,25 +14,37 @@
 from diff_gaussian_rasterization import GaussianRasterizationSettings, GaussianRasterizer
 from scene.gaussian_model import GaussianModel
 from utils.sh_utils import eval_sh
+from utils.general_utils import build_rotation
+import torch.nn.functional as F
+import matplotlib.pyplot as plt
 
-def render(viewpoint_camera, pc : GaussianModel, pipe, bg_color : torch.Tensor, scaling_modifier = 1.0, override_color = None):
+def render(viewpoint_camera, pc : GaussianModel, pipe, bg_color : torch.Tensor, scaling_modifier = 1.0, override_color = None,
+           return_depth=False, return_normal=False, return_opacity=False):
     """
-    Render the scene. 
-    
-    Background tensor (bg_color) must be on GPU!
+    渲染场景： 将高斯分布的点投影到2D屏幕上来生成渲染图像
```

Comment:
wth?

Reply:
Sorry about that! I made a mistake by submitting the merge request by accident.